### PR TITLE
Merge wied03/savant_2.0.0

### PR DESCRIPTION
**Pull request by wied03**
PR details

### CODE REVIEW

File: build.savant [hunk 1]
```diff
@@ -64,10 +65,10 @@ groovyTestNG.settings.groovyVersion = "4.0"
...
 idea.settings.imlFile = "dependency-plugin.iml"
 idea.settings.moduleMap = [
-    "org.savantbuild:savant-core:2.0.0-RC.7"                  : "savant-core",
-    "org.savantbuild:savant-dependency-management:2.0.0-RC.7" : "savant-dependency-management",
-    "org.savantbuild:savant-io:2.0.0-RC.7"                    : "savant-io",
-    "org.savantbuild:savant-utils:2.0.0-RC.7"                 : "savant-utils"
+    ("org.savantbuild:savant-core:${savantVersion}".toString())                 : "savant-core",
+    ("org.savantbuild:savant-dependency-management:${savantVersion}".toString()): "savant-dependency-management",
+    ("org.savantbuild:savant-io:${savantVersion}".toString())                   : "savant-io",
+    ("org.savantbuild:savant-utils:${savantVersion}".toString())                : "savant-utils"
```
Comment (robotdan) [inline build.savant:71]: Is this required? 

We use this in `fusionauth-app` w/out using `.toString()`. 

```
idea.settings.moduleMap = [
    "io.fusionauth:fusionauth-plugin-api:${fusionauthVersion}": "fusionauth-plugin-api"
]
```

Comment (wied03) [reply build.savant]: Reset this back to the method without the toStrings

File: dependency-plugin.iml [hunk 2]
```diff
@@ -12,224 +12,225 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="groovy-4.0" level="application" />
```
Comment (robotdan) [inline dependency-plugin.iml:15]: `Groovy 4.0`?
- https://github.com/savant-build/savant-version/pull/2/files#r1777646852

Comment (wied03) [reply dependency-plugin.iml]: Sorry. missed. wrong convention. Fixed.

**Review by robotdan — APPROVED** [review]

**Pull request merged**